services/skus/controllers.go

```diff
@@ -956,14 +956,14 @@ func HandleIOSWebhook(service *Service) handlers.AppHandler {
 		// transaction info
 		txInfo, err := req.GetTransactionInfo(ctx)
 		if err != nil {
-			logger.Warn().Err(err).Msg("failed to get renewal info from message")
+			logger.Warn().Err(err).Msg("failed to get transaction info from message")
 			validationErrMap["invalid-renewal-info"] = err.Error()
```

Comment:
`validationErrMap` is mismatched as well

Reply:
resolved: https://github.com/brave-intl/bat-go/pull/1649/commits/9064cd764800c268686b31b2d3153500162fca31